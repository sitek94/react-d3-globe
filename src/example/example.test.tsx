import * as React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';

import { Example } from './example';

describe('Example', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Example />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders correctly', () => {
    render(<Example />);

    expect(screen.getByText('Example')).toBeTruthy();
  });

  it(`renders correctly when passed "name" prop`, () => {
    render(<Example name="Maciek" />);

    expect(screen.getByText('Maciek')).toBeTruthy();
  });
});
