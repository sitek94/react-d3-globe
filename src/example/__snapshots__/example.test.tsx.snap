// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example should match the snapshot 1`] = `
<div>
  <h1>
    Example
  </h1>
  <p>
    test
  </p>
</div>
`;
